cms/publications/models.py

```diff
@@ -114,6 +116,29 @@ class PublicationPublicationTypeRelationship(models.Model):
     )
 
 
+class ChangelogEntry(models.Model):
+    change_date = models.DateTimeField(
+        "The time the changes were made", default=datetime.now
+    )
+    change_description = models.TextField("Description of the changes")
+
+    panels = [
+        FieldPanel("change_date"),
+        FieldPanel("change_description"),
+    ]
+
+    class Meta:
+        abstract = True
+
+
+class ChangelogEntryLink(ChangelogEntry):
```

Comment:
Some docstrings on the classes might help people in future, the decoupling is nice but not necessarily intuitively obvious.